compiler/gopherjspkg/fs.go

```diff
@@ -0,0 +1,31 @@
+// +build gopherjsdev
+
+package gopherjspkg
+
+import (
+	"go/build"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/shurcooL/httpfs/filter"
+)
+
+// FS is a virtual filesystem that contains core GopherJS packages.
+var FS = filter.Keep(
+	http.Dir(importPathToDir("github.com/gopherjs/gopherjs")),
+	func(path string, fi os.FileInfo) bool {
+		return path == "/" ||
+			path == "/js" || strings.HasPrefix(path, "/js/") ||
+			path == "/nosync" || strings.HasPrefix(path, "/nosync/")
```

Comment:
Not only `github.com/gopherjs/js`, but also `github.com/gopherjs/js/foobar` would be a special path?

Reply:
I made the filter more precise in 2ad4a5b, so it won't pick up any subdirectories now.